src/serlo-editor/math/math-help-modal.tsx

```diff
@@ -0,0 +1,84 @@
+import Modal from 'react-modal'
+
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
+export const MathHelpModal = ({
+  isHelpOpen,
+  setIsHelpOpen,
+}: {
+  isHelpOpen: boolean
+  setIsHelpOpen: (isOpen: boolean) => void
+}) => {
+  const mathStrings = useEditorStrings().plugins.text.math
+  return (
+    <Modal
```

Comment:
maybe follow up: using `ModalWithCloseButton` here would probably be a lot simpler.